ansible/roles/test/tasks/pytest_runner.yml

```diff
@@ -0,0 +1,64 @@
+- name: print a warning
+  debug:
+    msg:
+      - "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+      - "!!!!!! Ansible playbook for running {{ testcase_name }} is now deprecated !!!!!!"
+      - "!!!!!! This playbook is just a wrapper to run py.test in sonic-mgmt/tests !!!!!!"
+      - "!!!!!!!!!!!!!!!!!!!!!!!!!! Consider using py.test !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+      - "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+
+- fail:
+    msg: Test node is not defined
+  when: test_node is not defined
+
+- fail:
+    msg: Testbed is not defined
+  when: testbed_name is not defined
+
+- name: set py.test command variable
+  set_fact:
+    pytest_cmd: 'py.test {{ test_node }}'
+
+- name: append filter expression if needed
+  set_fact:
+    pytest_cmd: '{{ pytest_cmd }} -k "{{ test_filter }}"'
+  when: test_filter is defined
+
+- name: append mark if needed
+  set_fact:
+    pytest_cmd: '{{ pytest_cmd }} -m {{ test_mark }}'
+  when: test_mark is defined
+
+- name: append testbed name
+  set_fact:
+    pytest_cmd: '{{ pytest_cmd }} --testbed={{ testbed_name }}'
+
+# We could use ansible_inventory_sources magic variable but it is available in ansible >=2.5
+- name: append inventory file
+  set_fact:
+    pytest_cmd: '{{ pytest_cmd }} --inventory=../ansible/inventory'
+
+- name: append testbed file
+  set_fact:
+    pytest_cmd: '{{ pytest_cmd }} --testbed_file=../ansible/testbed.csv'
```

Comment:
can we get the testbed csv file name form the user input, not always this inventory file.

Reply:
For testbed.csv yes. I see some playbooks accept 'testbed_file' external variable I can use that as well.